### DISCUSSION TITLE
Choose which filters will be expanded in global

### DISCUSSION THREAD

**Discussion by Thorug** [Smile-SA/elasticsuite#2652] [Q&A] [answered]
Hello,
i know it is possible for each categories and each store choose display mode but i need set it in global for every categories though all stores and cannot find it.
Is there any way to do this?
Thank you!

**Comment by romainruaud** [comment, marked as answer]
Hi @Thorug 

not actually there is no way to configure "Always displayed" or "Always hidden" globally for filterable attributes.
